x-pack/solutions/observability/plugins/apm/public/components/app/settings/apm_indices/index.tsx

```diff
@@ -130,6 +114,8 @@ export function ApmIndices() {
             'The indices changes were successfully applied. These changes are reflected immediately in the APM UI',
         }),
       });
+      // Defer reload once the UI has finished rendering
+      setTimeout(() => window.location.reload());
```

Comment:
do we need to `clearCache();` here?

Reply:
`clearCache` was for the `callApmApi` machinery. `apmSourcesAccess` does not have any of that so there's no need for this particular instance.